example/schema/example.js

```diff
@@ -118,19 +118,6 @@ export const exampleDataType = parseField(DATA, {
   }
 });
 
-/*
```

Comment:
Ooo I see, well we have a record of them in source control if we ever want to add them back.